CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-health ChangeLog
 
+## 1.1.0 - TBD
+
+### Added
+- Do not listen for HTTP/HTTPS requests until readiness checks have passed. This
+  is compatible with deployment platforms that lack explicit health and
+  readiness checks, but instead expect that the application is ready when it
+  responds to requests sent to the HTTP/HTTPS server's TCP port.
+  - Add handler for the `bedrock-server.readiness-check` event.
```

Comment:
Was this intentionally indented? If not, here's a suggestion to reduce it:

```suggestion
- Add handler for the `bedrock-server.readiness-check` event.
```

Reply:
Yes it was to denote that this feature was in direct connection with the lengthy prose above.